daft/daft/__init__.pyi

```diff
@@ -1051,6 +1051,7 @@ class PyExpr:
     def approx_count_distinct(self) -> PyExpr: ...
     def approx_percentiles(self, percentiles: float | list[float]) -> PyExpr: ...
     def mean(self) -> PyExpr: ...
+    def stddev(self) -> PyExpr: ...
```

Comment:
Probably can add later if needed, but should we support an optional `ddof` parameter?

Reply:
Ah okay, this is a good feature to have. Will schedule it for a later release.